tensorflow/python/util/protobuf/compare.py

```diff
@@ -72,6 +71,8 @@ def testXXX(self):
 from google.protobuf import message
 from google.protobuf import text_format
 
+from tensorflow.python.util.compat import collections_abc
+
```

Comment:
tensorflow/python/util/protobuf/compare.py", line 75, in <module>
    from tensorflow.python.util.compat import collections_abc
ImportError: No module named tensorflow.python.util.compat

Reply:
I've replaced this line with a relative import, and fixed some merge conflicts.